docs/class-definitions.md

```diff
@@ -0,0 +1,73 @@
+## Class Definitions
+
+**Contents**
+
+- [Spaces](#spaces)
+- [SpaceOptions](#spaceoptions)
+- [Space](#space)
+- [SpaceMember](#spacemember)
+
+### Spaces
+
+#### constructor(ably)
+
+Create a new instance of the Spaces library.
+This requires a promise instance of the Ably library to be passed in.
+Refer to the [Ably-JS Documentation](https://github.com/ably/ably-js#introduction) for information on setting up a realtime promise client.
+
+| Property | Type                 |
+|----------|----------------------|
+| ably     | Ably.RealtimePromise |
+
+
+#### get(name, options?)
+Get or create a space. Options may only be provided if a space does not yet exist.
```

Comment:
space instance?

Reply:
Depends if we want an individual space to be referred to as a "Space" or a "Space Instance" in our terminology for space concepts

Reply:
Let's go with 'space instance' for now and we can review all terminology in another round (with DevEd and PMM)

Reply:
Updated the wording: https://github.com/ably-labs/spaces/pull/24/commits/a3a7ef11de3bd9cb06c6f7bd62c173787d1f5836